pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,18 +4,19 @@ title: Reduce bandwidth usage
 
 # Reduce bandwidth usage
 
-In order to provide our users with a clear view of their devices, the {{ $names.company.lower }} device supervisor constantly keeps our API informed about the devices' conditions and applies any changes, like downloading new application updates or environment variable changes by negotiating with our API. While a quick reflection of actions is great during development, it comes at a cost of increased data usage.
+In order to provide users with a wealth of information about their devices, the device's {{ $names.company.lower }} Supervisor regularly keeps the API informed about device state. Upon receiving target device state that differs from current device state, the Supervisor applies changes, such as downloading new application updates or setting environment variables. While a quick reflection of actions is great during development, it comes at a cost of increased data usage.
```

Comment:
```suggestion
In order to provide users with essential information about their devices, the {{ $names.company.lower }} Supervisor on the device regularly keeps the API informed about the device state. Upon receiving a target device state that differs from the current device state, the Supervisor applies changes, such as downloading new application updates or setting environment variables. While a quick response to actions is great during development, it also leads to increased data usage.
```

Reply:
I like Vipul's suggested edit

Reply:
Changed with the exception of 
- `regularly keeps the API informed about the device state` --> `regularly keeps the API informed about the device's state`. 
- `a target device state that differs from the current device state` --> `a target device state which differs from the current device state`